docs/02-要件定義.md

```
@@ -0,0 +1,94 @@
+ユーザー要望をもとに、機能要件・非機能要件を整理し、初期バージョンでのスコープを明確化します。
+
+## 2. 対象範囲
+- 画像形式：PNG／JPEG  
+- 認証：管理者が会員情報を登録し、会員のみがダウンロード可能  
+- ウォーターマーク：会員ID（例 `user001`）を斜め45°で全体に埋め込む  
+- ダウンロード：共有リンク不可。エンドポイント経由でのみ配信  
+- ストレージ：Supabase Storage（バケット無制限保存）  
+- Edge Function：Deno＋ImageScriptで動的にウォーターマーク生成  
+
+## 3. 用語定義
+| 用語           | 意味                                        |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)